dev/tasks/verify-rc/github.linux.amd64.yml

```diff
@@ -59,10 +59,6 @@ jobs:
           distribution: 'temurin'
           java-version: '11'
 
-      - uses: actions/setup-node@v2
-        with:
-          node-version: '16'
```

Comment:
This `.yml` is used only for `verify-rc-binaries-*-linux-amd64` and they don't use Node.js.